cernopendata_client/cli.py

```diff
@@ -100,25 +102,51 @@ def get_file_locations(server, recid, doi, title, protocol, expand):
     if recid is not None:
         validate_recid(recid)
     record_json = get_record_as_json(server, recid, doi, title)
-    file_locations = [file["uri"] for file in record_json["metadata"]["files"]]
-    if expand:
-        # let's unwind file indexes
-        file_locations_expanded = []
-        for file in file_locations:
-            if file.endswith("_file_index.txt"):
-                url_file = file.replace("root://eospublic.cern.ch/", server)
-                req = requests.get(url_file)
-                for url_individual_file in req.text.split("\n"):
-                    if url_individual_file:
-                        file_locations_expanded.append(url_individual_file)
-            elif file.endswith("_file_index.json"):
-                pass
-            else:
-                file_locations_expanded.append(file)
-        file_locations = file_locations_expanded
-
-    if protocol == "http":
-        file_locations = [
-            f.replace("root://eospublic.cern.ch/", server) for f in file_locations
-        ]
+    file_locations = get_files_list(server, record_json, protocol, expand)
     click.echo("\n".join(file_locations))
+
+
+@cernopendata_client.command()
+@click.option("--recid", type=click.INT, help="Record ID")
+@click.option("--doi", help="Digital Object Identifier.")
+@click.option("--title", help="Record title")
+@click.option(
```

Comment:
Since we don't have `root` supported yet, maybe don't introduce this option... or, better, switch the defaults to use `http` and if people select `root` then reply that this is not yet supported and do `sys.exit(1)`.

Reply:
(we could also with default to 'http' over 'root' for the other `get-file-locations` function)